src/components/Apps.js

```diff
@@ -1,14 +1,38 @@
 import React, { Component } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
-//import calculate from '../logic/calculate';
+import calculate from '../logic/calculate';
 
 class Apps extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      total: null,
+      next: null,
+      operation: null
+    }
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick = (buttonName) => {
+    const { total, next, operation } = calculate(this.state, buttonName);
```

Comment:
TIP:

- when you use arrow function syntax, you don't necessarily need to bind `this` in the constructor.